src/OrchardCore/Orchard.Indexing.Abstractions/IndexingTask.cs

```diff
@@ -10,9 +10,24 @@ public enum IndexingTaskTypes
 
     public class IndexingTask
     {
+        /// <summary>
+        /// The unique identifier of the <see cref="IndexingTask"/>.
+        /// </summary>
         public int Id { get; set; }
+
+        /// <summary>
+        /// The id of the content item that is represented by the <see cref="IndexingTask"/> instance.
+        /// </summary>
         public string ContentItemId { get; set; }
-        public DateTimeOffset CreatedUtc { get; set; }
+
+        /// <summary>
+        /// The <see cref="DateTime"/> value the task was created.
+        /// </summary>
+        public DateTime CreatedUtc { get; set; }
```

Comment:
I was passing a Datetime in the sql query parameters, and it failed on sqlite. So I changed the property. DateTimeOffset has no advantage over DateTime when we store them. And we explicitly use Utc.